tcms/testplans/static/testplans/js/get.js

```diff
@@ -1,5 +1,6 @@
 $(document).ready(function() {
-    const testPlanId = $('#test_plan_pk').data('testplanPk');
+    // declare it as global so all functions can access it
+    testPlanId = $('#test_plan_pk').data('testplanPk');
```

Comment:
how about passing it to the functions that need it? IMO it's better to avoid globals as much as possible. This way all functions will be self-contained and pure, because they will receive all its input as arguments, rather than globals. this will ease our work in writing unit tests, once we start unit-testing the js code.

as the saying goes:
Q: what's the best prefix for global variables?
A: `//`

Reply:
+1 for making it a parameter to other functions